prov/util/src/util_mem_monitor.c

```diff
@@ -171,15 +213,14 @@ void ofi_monitors_init(void)
 	pthread_mutex_init(&mm_state_lock, NULL);
 	pthread_rwlock_init(&mm_list_rwlock, NULL);
 
-	uffd_monitor->init(uffd_monitor);
-	memhooks_monitor->init(memhooks_monitor);
-	cuda_monitor->init(cuda_monitor);
-	cuda_ipc_monitor->init(cuda_ipc_monitor);
-	rocr_monitor->init(rocr_monitor);
-	rocr_ipc_monitor->init(rocr_ipc_monitor);
-	xpmem_monitor->init(xpmem_monitor);
-	ze_monitor->init(ze_monitor);
-	import_monitor->init(import_monitor);
+	initialize_monitor_list();
+
+	for (size_t i = 0; i < monitor_list_size; i++) {
+		FI_DBG(&core_prov, FI_LOG_MR,
```

Comment:
This could be FI_INFO instead of debug, and maybe assert that name is non-null?  Cleanup could also be FI_INFO

Reply:
I implemented the changes suggested by @shefty